launchable/plugin.py

```diff
@@ -44,8 +44,8 @@ def prepareTest(self, test):
 
             reorder(test, order)
 
-            self._print("Test execution optimized by Launchable 🚀")
-            return
+            self._print("Test execution optimized by Launchable")
```

Comment:
Why did you remove 🚀 ? 🤔

Reply:
Oh, sorry I was intended to leave a comment here and totally forgot about it!
I was reported that the emoji does not work in our customer's environment but I think we can try using Unicode instead. Let me try!